src/store/modules/target.js

```diff
@@ -5,6 +5,7 @@ export default ({
       {
         id: 1,
         name: 'My target 1',
+        transformedName: 'my-target-1',
```

Comment:
we are not going to have transformedName only name 

if we try to save a Target with the name 'My target 1' we convert that to 'my-target-1' either in the frontend or the backend and we'll save in the column 'Name' inside the DB the value 'my-target-1'.